texastribune/donations
# Create Business Membership wall

#### What's this PR do?
- Creates a generic wall component
- Creates Circle-specific and BusMem-specific wall components that customize the generic component
- Slightly changes the wall CSS (left-aligned text and headers) so that it works for both wall types
- Updates some heading levels

#### Why are we doing this? How does it help us?
Request from Catherine & April.

#### How should this be manually tested?
- Go to `/business` and confirm a wall loads near the bottom with three groups and links to each business name
- Go to `/circle` and confirm the wall loads

#### How should this change be communicated to end users?
We'll tell Catherine & April when it's live.

#### Are there any smells or added technical debt to note?
The BusMem and Circle JSON formats are slightly different. It means I have to use props and scoped slots to sort out some of the formatting differences. But most of the logic is shared and lives in the `Wall.vue` file.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/736178/todos/3872814770

#### Have you done the following, if applicable:
* [x] Checked accessibility? (it's not perfect, but I did update the heading hierarchy on the Circle and BusMem pages)

#### TODOs / next steps
* [x] Check with art team
* [x] Confirm with Daniel that JSON is accurate
* [x] Inform Catherine & April about process
* [x] Document in wiki
